docs/source/deduplication.rst

```diff
@@ -0,0 +1,101 @@
+System Fingerprints
+===================
+A system fingerprint is composed of a set of system facts, products, entitlements, sources, and a metadata. For illustration a sample is included. ::
```

Comment:
, and metadata. The following example shows a system fingerprint.

Reply:
RE the comment about **exactly** what is a fact (below), also need to define **exactly** what is a fingerprint. This whole chunk of output? Each section in the output?